src/renderer/components/monaco-editor/monaco-theme.clouds-midnight.json

```diff
@@ -1,4 +1,5 @@
 {
```

Comment:
I think it would be better if we kept the folder `monaco-themes` instead of prefixing the files.

Reply:
Okay, but in general I prefer to have some some key text in a filename to make search by files easier in project..